services/skus/service.go

```diff
@@ -536,43 +547,29 @@ func (s *Service) CreateOrderFromRequest(ctx context.Context, req model.CreateOr
 				return nil, fmt.Errorf("failed to create checkout session: %w", err)
 			}
 
-			err = s.Datastore.AppendOrderMetadata(ctx, &order.ID, "stripeCheckoutSessionId", session.SessionID)
-			if err != nil {
-				return nil, fmt.Errorf("failed to update order metadata: %w", err)
-			}
-
-		case order.IsRadomPayable():
```

Comment:
`CreateOrderFromRequest` is no longer used for Premium orders, so there is no way code for handling Radom is needed here.

The Stripe bit remains though as there might be lots of legacy (and useless) tests using it.